src/Validation/Validation.php

```diff
@@ -1463,12 +1463,12 @@ public static function imageSize($file, array $options): bool
     /**
      * Validates the image width.
      *
-     * @param array $file The uploaded file data from PHP.
```

Comment:
Could we just be concrete here with array|UploadedFile ?
That would help also understanding of IDE/PHPStan etc.

Reply:
All other validation methods also have `mixed` since the argument comes from user input which can be of any type.

Reply:
But then you need to guard it with a type check and early return false
Otherwise, semantically, this can blow up from user land site as internal error without it supposed to happen.

Reply:
> But then you need to guard it with a type check and early return false

Which is done by the `Valdiation::getFilename()` method which is called further down the call chain.